packages/zowe-explorer-api/src/profiles/ProfilesCache.ts

```diff
@@ -391,20 +347,49 @@ export class ProfilesCache {
         };
     }
 
-    // used by Zowe Explorer for v1 profiles
-    protected async deleteProfileOnDisk(profileInfo: zowe.imperative.IProfileLoaded): Promise<void> {
-        await this.getCliProfileManager(profileInfo.type).delete({ name: profileInfo.name });
-    }
-
-    // used by Zowe Explorer for v1 profiles
-    protected async saveProfile(profileInfo: Record<string, unknown>, profileName: string, profileType: string): Promise<zowe.imperative.IProfile> {
-        const newProfile = await this.getCliProfileManager(profileType).save({
-            profile: profileInfo,
-            name: profileName,
-            type: profileType,
-            overwrite: true,
+    public async convertV1ProfToConfig(): Promise<ZeApiConvertResponse> {
+        const successMsg: string[] = [];
+        const warningMsg: string[] = [];
+        const zoweDir = FileManagement.getZoweDir();
+        const profilesPath = path.join(zoweDir, "profiles");
+        const oldProfilesPath = `${profilesPath.replace(/[\\/]$/, "")}-old`;
+        const convertResult = await zowe.imperative.ConfigBuilder.convert(profilesPath);
+        for (const [k, v] of Object.entries(convertResult.profilesConverted)) {
+            successMsg.push(`Converted ${k} profiles: ${v.join(", ")}\n`);
```

Comment:
I think it would be better to remove the commas. On Windows:
![image](https://github.com/zowe/vscode-extension-for-zowe/assets/122040687/f4c02c03-db54-4e16-8788-41c401168825)

Reply:
Sorry @JillieBeanSim, I mistakenly assumed this line was adding the commas.

Instead, down where it returns the strings I think it should be something like:
```
        return {
            success: String(successMsg.join('')),
            warnings: String(warningMsg.join('')),
            convertResult,
        };
```

Reply:
pushed again after testing on a VM
![Screenshot 2024-02-12 at 9 50 36 AM](https://github.com/zowe/vscode-extension-for-zowe/assets/49491949/c85cf2e7-b786-43c1-bd68-a53456819f43)